app/org/sagebionetworks/bridge/exceptions/NotImplementedException.java

```diff
@@ -0,0 +1,21 @@
+package org.sagebionetworks.bridge.exceptions;
+
+import org.apache.http.HttpStatus;
+
+/**
+ * This endpoint has not been implemented for this study. It has been disabled, or 
+ * not completely configured. The caller cannot fix the call to succeed, but it is 
+ * not an unexpected server error, it is intentional.
+ */
+@SuppressWarnings("serial")
+public class NotImplementedException extends BridgeServiceException {
```

Comment:
Do you think the SDK should get a new exception as well? Or roll it up into BridgeSDKException?